values.yaml

```diff
@@ -834,6 +838,7 @@ server:
       # https://www.vaultproject.io/docs/platform/k8s/helm/run#protecting-sensitive-vault-configurations
       config: |
         ui = true
+        cluster_name = {{ .Values.cluster_name | default (printf "%s-k8s" (include "vault.name" .)) | quote }}
```

Comment:
Does this actually work? I've never seen the templates used outside of the K8s yaml helm is generating.

Reply:
Yes it works - I've tested it (with both explicit value set & nothing set) before making PR

Reply:
Example as per current `helm` documentation.
https://helm.sh/docs/chart_template_guide/functions_and_pipelines/#using-the-default-function